gradle.properties

```diff
@@ -1,5 +1,8 @@
 # JVM args to run gradle
-org.gradle.jvmargs=-Xms1024m -Xmx2048m
+org.gradle.jvmargs=-Xms1024m -Xmx2048m \
+    -Djdk.xml.maxGeneralEntitySizeLimit=0 \
```

Comment:
Why does the gradle process need these properties? I can see that the test processes need them.

Reply:
I will check again, but I remember I had to them very early in the process, before I got to the tests.